broker/service/schema_service.py

```diff
@@ -0,0 +1,22 @@
+import ast
+import json
+import jsonref
+
+
+class SchemaService(object):
+    def __init__(self):
+        self.json_loader = jsonref.JsonLoader()
+
+    def dereference_schema(self, json_schema):
+        json_ref_obj = jsonref.loads(json.dumps(json_schema), loader=self.json_loader)
+        return json_ref_obj
```

Comment:
Probably not a good idea to return an object type from a third party library. Might be best to keep usage of `jsonref` internal so it's easy to replace it with something if ever.